docs/source/index.rst

```diff
@@ -154,6 +154,11 @@ Link to Cleanlab Studio docs: `help.cleanlab.ai <https://help.cleanlab.ai/>`_
    Datalab Tutorials <tutorials/datalab/index>
```

Comment:
please rebase to make this look like the master branch version of the index, except for inserting your new `Improving ML Performance` tutorial in the sidebar right in between `Datalab Tutorials` and `CleanLearning Tutorials`. I think that's the best place for it (it's technically a datalab tutorial too, but is so important I think it should be outside of the the Datalab sidebar)

Reply:
Yep just made sure my version is the master branch version with my tutorial added in between datalab and cleanlearning